### ISSUE TITLE
LaTeX Support

### ISSUE THREAD

**Issue by arcticicestudio** [arcticicestudio/northem-light-atom-syntax#6]
This is a request I've received via :email: email.
Added <img src="http://www.latex-project.org/favicon.ico" width=16 height=16/> [LaTeX](https://atom.io/packages/language-latex)(`.tex`,`.bib`, `.bbx`, `.cbx`, `.sty`) to the [backlog](https://github.com/arcticicestudio/northem-light-atom-syntax/issues/5) list.

**Comment by arcticicestudio** [comment]
Released version [`0.19.0`](https://github.com/arcticicestudio/northem-light-atom-syntax/releases/tag/v0.19.0) including LaTeX support.
Please feel free to post feedback or improvement suggestions.

**Comment by zhrkevin** [comment]
Here are some error occurs when the following steps are taken:

[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.7.3
**System**: Mac OS X 10.11.4
**Thrown From**: [northem-light-atom-syntax](https://github.com/arcticicestudio/northem-light-atom-syntax) package, v0.19.0

### Stack Trace

Failed to activate the northem-light-atom-syntax theme

```
At Unrecognised input in /Users/zhengren/.atom/packages/northem-light-atom-syntax/styles/northem-light.less:159:1

LessError: Unrecognised input
  at /Users/zhengren/.atom/packages/northem-light-atom-syntax/styles/northem-light.less:159:1
```

### Commands

```
```

### Config

``` json
{
  "core": {
    "disabledPackages": [
      "dvnk-icons",
      "latex-autocomplete",
      "latextools",
      "latex-plus"
    ],
    "themes": [
      "northem-light-atom-ui",
      "northem-light-atom-syntax"
    ]
  }
}
```

### Installed Packages

``` coffee
# User
color-picker, v2.1.1
file-icons, v1.7.9
grammar-token-limit, v0.1.1
highlight-selected, v0.11.2
language-latex, v0.6.1
latexer, v0.3.0
linter, v1.11.4
linter-chktex, v1.3.0
minimap, v4.23.4
minimap-bookmarks, v0.3.0
minimap-highlight-selected, v4.4.0
minimap-pigments, v0.2.0
northem-light-atom-ui, v0.4.0
pdf-view, v0.50.0
pigments, v0.26.2
unfancy-file-icons, v0.6.0

# Dev
No dev packages
```

**Comment by arcticicestudio** [comment]
Released a new patch version [`v0.19.1`](https://github.com/arcticicestudio/northem-light-atom-syntax/releases/tag/v0.19.1) to fix the "Unrecognized input" exception.
Please update your package via `apm`.
